.travis.yml

```diff
@@ -112,11 +112,6 @@ jobs:
     env:
       TOXENV: lint
 
-  - <<: *lint
```

Comment:
I'm "-1" here: collapsing multiple separate distinct checks into one results in the undebuggable log. We already have this problem, no need to make it worse.